gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -275,6 +278,18 @@ public GsonBuilder enableComplexMapKeySerialization() {
     return this;
   }
 
+  /**
+   * Configure Gson to deserialize duplicate map key. By default, Gson throw a {@code JsonSyntaxException} when there are more than
+   * one same key.
```

Comment:
Maybe describe in more detail how duplicate keys will be treated:
```suggestion
   * Configures Gson to deserialize duplicate map keys. Only the value of last entry with the same key will be used, previous values
    * will be discarded. By default, Gson throws a {@link JsonSyntaxException} when a key occurs more than once.
```